File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/WinAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class WinAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playWinAnimation() {
        new AnimationTester().testAnimation(new WinAnimation(), 100);
    }
}

File: test/hud/animations/WinAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {WinAnimation} from '../../../src/hud/animations/WinAnimation.js';

/**
 * Integration test for victory animation on actual hardware.
 */
describe('WinAnimation Integration', () => {
    test('playWinAnimation', async () => {
        await new AnimationTester().testAnimation(new WinAnimation(), 100);
    }, 30000);
});
